### PR TITLE
[16.0][MIG] mrp_flattened_bom_xlsx_direct_materials_cost: Migration to 16.0

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

**Comment by PauBForgeFlow** [comment]
Depends on [PR#112](https://github.com/OCA/manufacture-reporting/pull/112)

**Review by BernatPForgeFlow — APPROVED** [review]
Functional and code review OK!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
@PauBForgeFlow @BernatPForgeFlow can you review the dependency then?

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo-addon-mrp_flattened_bom_xlsx@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/112/head#subdirectory=setup/mrp_flattened_bom_xlsx
+odoo-addon-report_xlsx@git+https://github.com/OCA/reporting-engine.git@16.0#subdirectory=setup/report_xlsx
```
Comment (LoisRForgeFlow) [inline test-requirements.txt]: @PauBForgeFlow  Can you remove the test-requirements and rebase this PR? dependencies are merged.

Comment (PauBForgeFlow) [reply test-requirements.txt]: Done

**Pull request merged**